### PR TITLE
OSDOCS--2572: Machine management overview page

### PR DESCRIPTION
**Pull request by Amrita42**
Applies to 4.6+ versions

Create an overview page for machine management.

[OSDOCS-2572](https://issues.redhat.com/browse/OSDOCS-2572)   

[Preview](https://deploy-preview-38127--osdocs.netlify.app/openshift-enterprise/latest/machine_management/index.html)

@sunzhaohua2  please review .

### CODE REVIEW

**Review by EricPonvelle — APPROVED** [review]
/lgtm

Some minor comma nits.

File: machine_management/index.adoc [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+. xref:../machine_management/modifying-machineset.adoc#modifying-machineset[Modify a machine set] through the MachineSet YAML configuration file.
+. xref:../machine_management/deleting-machine.adoc#deleting-machine[Delete] a machine.
+. xref:../machine_management/creating-infrastructure-machinesets.adoc#creating-infrastructure-machinesets[Create infrastructure machine sets].
+. Configure and deploy a xref:../machine_management/deploying-machine-health-checks.adoc#deploying-machine-health-checks[machine health check] to automatically fix damaged machines in a machine pool.
+
+[discrete]
+== Autoscaler
+
+The autoscaler ensures that your cluster is flexible to changing workloads.
+To xref:../machine_management/applying-autoscaling.adoc#applying-autoscaling[autoscale] your {product-title} cluster, you must first deploy a cluster autoscaler, and then deploy a machine autoscaler for each machine type.
```
Comment (sunzhaohua2) [inline machine_management/index.adoc:48]: I think this should be `and then deploy a machine autoscaler for each **machine set**.`

**Comment by sunzhaohua2** [comment]
one comment, others /lgtm

File: machine_management/index.adoc (hunk 1 above)
Comment (Amrita42) [reply machine_management/index.adoc:48]: good catch! its resolved now.

**Pull request merged**